init.d/glastopf

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# This file is part of the Glastopf honeypot
+#
+# Main developer - Colleen Blaho <cblaho@sas.upenn.edu>
+# Based on the Kojoney2 script by - Justin C. Klein Keane <jukeane@sas.upenn.edu>
+# And Jose Antonio Coret <joxeankoret@yahoo.es>
+# Last updated 9 Jan 2015
+#
+# chkconfig: 2345 99 15
+# description: Glastopf, the web application honeypot -- http://glastopf.org
+#
+# processname: glastopf
+# config: $workdir/glastopf.cfg
+# pidfile: none
+
+
+workdir=/opt/myhoneypot
```

Comment:
I'd pass this in as a parameter and in case the parameter is missing, ask if `/opt/myhoneypot` should be used.